code/game/machinery/vending/vendor_types/crew/senior_officers.dm

```diff
@@ -268,6 +268,67 @@ GLOBAL_LIST_INIT(cm_vending_clothing_cmo, list(
 
 
 //------------ EXECUTIVE OFFFICER ---------------
+
+//------------WEAPON VENDOR---------------
+GLOBAL_LIST_INIT(cm_vending_gear_executive_officer, list(
+		list("CAPTAIN'S PRIMARY (CHOOSE 1)", 0, null, null, null),
+		list("M41A MK1 Pulse Rifle", 0, /obj/item/storage/box/guncase/m41aMK1AP, MARINE_CAN_BUY_KIT, VENDOR_ITEM_MANDATORY),
+		list("MK221 Tactical Shotgun", 0, /obj/effect/essentials_set/xo/shotgunpreset, MARINE_CAN_BUY_KIT, VENDOR_ITEM_MANDATORY),
+
+		list("PRIMARY AMMUNITION", 0, null, null, null),
+		list("M41A MK1 Magazine", 40, /obj/item/ammo_magazine/rifle/m41aMK1, null, VENDOR_ITEM_RECOMMENDED),
```

Comment:
40 points?

Reply:
Limits you to 3 additional mags. 

I'm open to changing prices